packages/deployer/subtasks/validate-initializables.js

```diff
@@ -0,0 +1,35 @@
+const { subtask } = require('hardhat/config');
+const mapValues = require('just-map-values');
+const logger = require('@synthetixio/core-js/utils/io/logger');
+const ModuleInitializableASTValidator = require('../internal/initializable-ast-validator');
+const { ContractValidationError } = require('../internal/errors');
+const { SUBTASK_VALIDATE_INITIALIZABLES } = require('../task-names');
+
+subtask(SUBTASK_VALIDATE_INITIALIZABLES).setAction(async (_, hre) => {
+  logger.subtitle('Validating initializable contracts');
+
+  const { deployment } = hre.deployer;
+
+  const asts = mapValues(deployment.sources, (val) => val.ast);
+  const validator = new ModuleInitializableASTValidator(asts);
+
+  let errorsFound = [];
+  errorsFound.push(...validator.findMissingInitializer());
+  errorsFound.push(...validator.findMissingIsInitialized());
+
+  if (errorsFound.length > 0) {
+    errorsFound.forEach((error) => {
+      logger.error(error.msg);
+    });
+
+    if (logger.debugging) {
+      errorsFound.map((err) => console.log(JSON.stringify(err, null, 2)));
```

Comment:
Should use the `logger.debug` fn for output consistency:
```suggestion
      errorsFound.map((err) => logger.debug(JSON.stringify(err, null, 2)));
```

Reply:
implemented